testsuite/MDAnalysisTests/analysis/test_encore.py

```diff
@@ -444,6 +445,8 @@ def test_clustering_two_ensembles(self, ens1, ens2):
         assert len(cluster_collection) == expected_value, "Unexpected " \
                                                           "results: {0}".format(cluster_collection)
 
+    @pytest.mark.xfail(platform.machine() == "arm64" and platform.system() == "Darwin",
+                       reason="Fails on M1 Mac")
```

Comment:
could also reference the GH issue proper here, I just didn't search for it until after I opened the PR

Reply:
If possible that'd be great.